src/home/templates/partials/_table.html

```diff
@@ -1,3 +1,9 @@
+<head>
+    {% load static %}
+     <!-- Custom styles -->
```

Comment:
nit: comment not needed. (not a blocker)

Reply:
Addressed in most recent commit